### PR TITLE
refactor parameter names of text jobs

### PR DESCRIPTION
**Pull request by JackTemaki**

### CODE REVIEW

File: text/processing.py [hunk 1]
```diff
@@ -10,7 +10,12 @@ class PipelineJob(Job):
     """
 
     def __init__(
-        self, text, pipeline, zip_out=False, check_equal_length=False, mini_task=False
+        self,
+        text_file,
+        pipeline,
+        zip_out=False,
```
Comment (curufinwe) [inline text/processing.py:16]: better: zip_output

**Review by curufinwe — CHANGES_REQUESTED** [review]

File: text/processing.py [hunk 2]
```diff
@@ -10,7 +10,12 @@ class PipelineJob(Job):
     """
 
     def __init__(
-        self, text, pipeline, zip_out=False, check_equal_length=False, mini_task=False
+        self,
+        text_file,
```
Comment (michelwi) [inline text/processing.py:14]: also rename `text` to `text_file` below in lines 22 and 28 

also the run method needs to be modified

**Review by michelwi — CHANGES_REQUESTED** [review]

File: text/processing.py [hunk 3]
```diff
@@ -11,39 +11,40 @@ class PipelineJob(Job):
...
         pipeline,
-        zip_out=False,
+        zip_output=False,
         check_equal_length=False,
         mini_task=False,
     ):
         """
-
-        :param Path text: text file (raw or gz) to be processed
+        :param iterable[Path]|Path text_file: text file (raw or gz) or list of files to be processed
```
Comment (curufinwe) [inline text/processing.py:21]: text_file -> text_files

File: text/processing.py [hunk 4]
```diff
@@ -70,19 +71,19 @@ def tasks(self):
...
     def run(self):
         pipeline = self.pipeline.copy()
-        if self.zip_out:
+        if self.zip_output:
             pipeline.append("gzip")
         self.pipe = " | ".join([str(i) for i in pipeline])
-        if isinstance(self.text, (list, tuple)):
-            self.input_text = " ".join(gs.file_caching(str(i)) for i in self.text)
+        if isinstance(self.text_files, (list, tuple)):
+            self.input_text = " ".join(gs.file_caching(str(i)) for i in self.text_files)
```
Comment (curufinwe) [inline text/processing.py:78]: do not set new member var in run function

File: text/processing.py [hunk 5]
```diff
@@ -70,19 +71,19 @@ def tasks(self):
...
         else:
-            self.input_text = gs.file_caching(str(self.text))
-        self.sh("zcat -f {input_text} | {pipe} > {out}")
+            self.input_text = gs.file_caching(str(self.text_files))
+        self.sh("zcat -f {text_file} | {pipe} > {out}")
```
Comment (curufinwe) [inline text/processing.py:81]: here the input_text here refered to a member variable, now would be a new local var, probably better to insert via explicit string formatting

File: text/processing.py [hunk 6]
```diff
@@ -70,19 +71,19 @@ def tasks(self):
...
 
         # assume that we do not want empty pipe results
         assert not (os.stat(str(self.out)).st_size == 0), "Pipe result was empty"
 
-        input_length = int(self.sh("zcat -f {input_text} | sed '$a\\' | wc -l", True))
+        input_length = int(self.sh("zcat -f {text_file} | sed '$a\\' | wc -l", True))
```
Comment (curufinwe) [inline text/processing.py:86]: see above

**Review by curufinwe — CHANGES_REQUESTED** [review]

**Review by michelwi — APPROVED** [review]

**Review by curufinwe — APPROVED** [review]

**Pull request merged**